lib/main_common.pm

```diff
@@ -1126,6 +1126,7 @@ sub load_consoletests {
         loadtest "console/xorg_vt";
     }
     loadtest "console/zypper_lr";
+    loadtest "console/check_system_info" if (is_sle && (get_var('SCC_ADDONS') !~ /ha/) && !is_sles4sap && (is_upgrade || get_var('MEDIA_UPGRADE')));
```

Comment:
I think this can be safely scheduled for SLES4SAP as I don't believe we have any test using variable `MILESTONE_VERSION` and without the module only check for files that should exist in SLES4SAP. WDYT @ldevulder